tests/virtualization/virtman_view.pm

```diff
@@ -50,7 +51,12 @@ sub run {
     assert_screen 'virt-manager';
 
     # go to view now
-    wait_screen_change { send_key 'alt-v' };
+    if (is_aarch64) {
```

Comment:
Why this has to be specific to aarch64?.

Reply:
because it failed on aarch64 if  I remove wait_screen_change here:
http://f40.suse.de/tests/4750#step/virtman_view/9

Reply:
that is no reason, just an observation ...

Reply:
updated, please check, thanks.